on-off-board/onboarding.md

```diff
@@ -4,77 +4,96 @@ Welcome to NYPL Digital!
+4. You'll start meeting with your team members and collaborators to start planning your work
 
-### Commonly used accounts
+After that your intro will vary depending on your team and role, but it should focus at least in these areas
 
-A developer will generally need accounts for the following services:
+## 1. Learning
 
-- Slack
-  - [nypl.slack.com](https://nypl.slack.com/)
-- GitHub (Can use a personal account or make a new NYPL specific GitHub account)
+There is a lot to learn in Digital, and there is no way to pick everything up in your first month, let alone your first day! So here are some tips for starting the process of learning the whats, wheres, hows and whys of our department.
+
+- Review the [Contents](../README.md#contents) of this repo.
+- Review READMEs (or Wikis) of the repositories your team maintains
+  - Your manager and onboarding buddy should help give you context for them
+  - A good approach is to spin up local dev environments for any repositories you'll be contributing to.
+- Set up meetings with other engineers on your team to get a sense for the status of the team's work and where you can help contribute!
+- Review the documents and initial tasks outlined in the specific onboarding documents provided to you on your first day (Which is probably how you ended up here!)
+- Ask questions. If your manager did not ask you to do this already: start a document with questions and things that were unclear!
+  - This document should be an important part of your early 1:1s with your manager
+  - We want to use these questions to improve our documentation and processes! People with fresh eyes often see things that we miss
+
+## 2. Account Setup
+
+### Password Management
+
+We __strongly__ encourage team members to use a password manager for their credentials. There will be many passwords you need to track and the dangers of password reuse are real. We recommend one of the following password management tools:
+
+- [Keeper](https://keepersecurity.com)
+  - This is NYPL's approved password manager
+  - To create a keeper account attempt to sign in with your `@nypl.org` email address, you should be prompted to request Admin approval.
+- [MacPass](https://macpassapp.org/)
+  - This is a local, open source, password management solution
+  - It is not officially supported, but provides an easy place to track passwords
 
-  - NYPL GitHub organization:
+### Commonly used accounts
 
-    - [NYPL](https://github.com/NYPL)
-    - See [GitHub Organization Management](./github-org-management.md) for onboarding steps.
+A developer will generally need accounts for the following services. Most of these accounts will be configured for you during onboarding, please request access for any systems you do not have access to.
 
-  - Old and no-longer used organizations
+- [Slack](https://nypl.slack.com/) Preferred method of communication
+- GitHub (Can use a personal account or make a new NYPL specific GitHub account)
```

Comment:
maybe have a note here about how we ask for your name to appear in your github account somewhere so if you don't want your github account to be tied with your name you might want to make a new work account